src/components/main_elements/Twitter/TwitterFeed.jsx

```diff
@@ -0,0 +1,27 @@
+const TwitterFeed = ({ name, screen_name, url, profileImg, createdAt }) => {
+  return (
+    <>
+      <div className="p-[1rem]">
+        <div className="flex items-center gap-2">
+          <div className="">
+            <img
+              className="rounded-full w-12 sm:w-6"
+              src="https://i.pravatar.cc/300"
+            ></img>
+          </div>
+          <div className="flex items-baseline gap-1">
+            <h1 className="sm:text-[0.8rem] font-semibold">Amazon AWS</h1>
+            <p className="text-[0.8rem] text-gray">@amazonaws</p>
+          </div>
+        </div>
+        <div className="h-auto">
+          <p className="break-all">
+            Tweetaasdasdsadadasdasdaasdasdsdasdasdadasd
```

Comment:
??

Reply:
Hello abi, before bringing values with APIs, I prepared a design and set some fixed values. However, I forgot entry "Lorem Ipsum" instead of random values. In this week, I'll change these values into real data values instead of fixed values.